src/csharp/PesterTests/ContainerInfoTests.cs

```diff
@@ -0,0 +1,21 @@
+namespace PesterTests
+{
+    using Pester;
+    using Xunit;
+
+    /// <summary>
+    /// Container Info Tests
```

Comment:
Sorry, but those annotations are just noise.

Reply:
I'll silence the linter in other ways and remove the xml comments.